src/DurableTask.Netherite/Events/PartitionEvents/Internal/BatchProcessed.cs

```diff
@@ -10,6 +10,9 @@ namespace DurableTask.Netherite
     using DurableTask.Core;
     using DurableTask.Core.History;
 
+    /// <summary>
+    /// Update event that executes when a orchestration work item is completed.
+    /// </summary>
```

Comment:
based on my assumptions, I presume a _batch_ refers to multiple WIs.
```suggestion
    /// <summary>
    /// Update event that executes when a orchestration work items are completed.
    /// </summary>
```